terraform-e2e/README.md

```diff
@@ -4,6 +4,8 @@ This directory contains terraform configuration to be used for deploying verific
 
 ## Prerequisite
 
+Terraform version > 1.13.1
```

Comment:
We don't need to document this. If 1.13 is a requirement, we should add this to the terraform configs under `required_version` (we already have 1.13 there, just bump it). Better to capture this as code that docs.

Reply:
Added in code